indico/web/templates/forms/location_widget.html

```diff
@@ -1,34 +1,66 @@
+{# The location widget provides 3 core fields to describe the location; venue,
```

Comment:
Just a thought. This kind of specification should be done through tests rather than text. This docstring will become obsolete rather sooner than later.